### DISCUSSION TITLE
Are all the applications listed in the application page available in the latest version of Neurodesktop?

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#118] [Q&A] [answered]
Hi all,

At the top of the applications page:
https://neurodesk.github.io/applications/
I want to add something that says "If any of the applications is not available under _Neurodesk_ --> _All applications_ in Neurodesktop's start menu (notice that menu entries are sorted alphabetically), please re-install Neurodesktop following the platform-specific instructions at https://neurodesk.github.io/"
My question is -- are all the applications shown in https://neurodesk.github.io/applications/ indeed available in the menu of the latest Neurodesktop version? Or is it that things might be out-of-sync sometimes?

Oren

**Comment by stebo85** [comment, marked as answer]
yes, that's a good idea. The application list gets automatically generated based on the latest desktop release, so should always be up-to-date.

**Comment by civier** [comment]
@stebo85, Can I edit
https://neurodesk.github.io/applications/
like any other page of the documentation (choosing "edit this page" on the righthand panel), or will it ruin the search option?

**Reply by stebo85** [reply]
There is only one way to find out I am afraid ... I haven't tested this yet :D
